de_templates/test_config_ingest.yaml

```diff
@@ -0,0 +1,43 @@
+# `dataset_name` parameter is used to determine names of the following ADF resources:
+pipeline: Ingest_test
+# - dataset: ds_<dataset_name>
+# - linked service: ls_<dataset_name>
+dataset_name: AzureSql_Example
```

Comment:
I have previously removed the test config from my branch and then re-added, so I guess that is why it doesn't show the diff. Anyway, following @adurkan-amido's suggestion in the templating PR, I have removed the `pipeline_name` from this config. We decided that the pipeline will be named as `Ingest_{{ dataset_name }}`. Can you use the same for naming the pipeline?

Reply:
Done